package.json

```diff
@@ -6,8 +6,10 @@
   "type": "module",
   "scripts": {
     "start": "webpack serve --open",
+    "clean": "rimraf dist",
     "dev": "webpack serve --mode development",
     "build": "webpack --mode production",
+    "serve": "node server.js",
```

Comment:
As I understand, you deleted serve.js so please remove it also from here. No leftovers in code.